emission/analysis/modelling/trip_model/forest_classifier.py

```diff
@@ -0,0 +1,198 @@
+import pandas as pd
+from sklearn.preprocessing import OneHotEncoder
+import joblib
+from typing import Dict, List, Optional, Tuple
+from sklearn.metrics.pairwise import haversine_distances
+import emission.core.wrapper.confirmedtrip as ecwc
+import logging
+from io import BytesIO
+
+import emission.analysis.modelling.trip_model.trip_model as eamuu
+import emission.analysis.modelling.trip_model.config as eamtc
+import emission.storage.timeseries.builtin_timeseries as estb
+import emission.storage.decorations.trip_queries as esdtq
+from emission.analysis.modelling.trip_model.models import ForestClassifier
+
+EARTH_RADIUS = 6371000
+
+class ForestClassifierModel(eamuu.TripModel):
+
+    def __init__(self,config=None):
+
+        if config is None:
+            config = eamtc.get_config_value_or_raise('model_parameters.forest')
+            logging.debug(f'ForestClassifier loaded model config from file')
+        else:
+            logging.debug(f'ForestClassifier using model config argument')
+    
+        random_forest_expected_keys = [
+            'loc_feature',
+            'n_estimators',
+            'criterion',
+            'min_samples_split',
+            'min_samples_leaf',
+            'max_features',
+            'bootstrap',
+        ]            
+        cluster_expected_keys= [
```

Comment:
This is  is used when we cluster the coordinates (loc_cluster parameter = True) before passing to Random Forest .  However, Since we are  working with configuration where loc_cluster is `False` with RF , I'll mention the same there and  comment this and other related parts.